.circleci/docker/common/install_rocm.sh

```diff
@@ -142,13 +142,7 @@ install_centos() {
   if [[ $OS_VERSION == 9 ]]; then
       yum update -y --nogpgcheck
       dnf --enablerepo=crb install -y perl-File-BaseDir
-      yum install -y --nogpgcheck \
-                   rocm-dev \
-                   rocm-utils \
-                   rccl \
-                   rocprofiler-dev \
-                   roctracer-dev
-      yum install -y hipblas hipblas-devel hipcub-devel hipfft hipfft-devel hipsolver hipsolver-devel hipsparse hipsparse-devel miopen-hip miopen-hip-devel rccl rccl-devel rocalution rocalution-devel rocblas rocblas-devel rocfft rocfft-devel rocprim-devel rocrand rocrand-devel rocsolver rocsolver-devel rocsparse rocsparse-devel rocthrust-devel --nogpgcheck
+      yum install -y --nogpgcheck rocm-ml-sdk rocm-developer-tools
```

Comment:
@jithunnair-amd , this change is NOT backward compatible, it is available from ROCm 5.3 only.
For CentOS stream 9 we have support from ROCm 5.3 onwards, so updated this change. 
It helps in checking the new ROCm meta packages, so that we dont find problem later.